programmers/난이도별/level01.나누어_떨어지는_숫자배열/JiaLee0707.js

```diff
@@ -0,0 +1,4 @@
+function solution(arr, divisor) {
+    let answer = arr.filter((value) => value % divisor === 0);
+    return answer.length > 0 ? answer.sort((value1, value2) => value1 - value2) : [-1];
```

Comment:
```suggestion
    return answer.length ? answer.sort((value1, value2) => value1 - value2) : [-1];
```
자바스크립트에선 `> 0`을 작성해주지 않고 그냥 변수명만 입력을 해도 `undefined | null | 0`값을 체크할 수 있어요. 그래서 코드를 더 줄인다면 위처럼 줄일 수 있을 것 같아요. _(현재의 코드가 더 알아보기는 쉽지만요!)_
```suggestion
    return answer[0] ? answer.sort((value1, value2) => value1 - value2) : [-1];
```
추가로 만약 배열이 길 경우 (ex: 극단적으로, 계산해야 하는 숫자의 개수가 1억 개 이상) 배열 **첫 번째 방의 값 유무**로 판단을 하면 더 빨라질 수도 있을 것 같아 의견 남깁니다!

Reply:
오... 그런 방법도 있겠네요~

Reply:
ㅎㅎ 저도 검색해보니 궁금해져서 테스트를 진행해보았어요!

Reply:
아름님 덕분에 이번에는 자바스크립트의 특성에 대해 하나 알게 되었네요..!!
항상 좋은 리뷰 감사합니다😻